ansible/roles/bigbluebutton/defaults/main.yml

```diff
@@ -1,10 +1,12 @@
 ---
 
 galaxy_path: "{{ playbook_dir }}/roles.galaxy"
-# Git commit or release version of bbb_role_git
-bbb_role_ver: 0104e9eefcaf88b9129a287953984fcaa63adc41
-# https://github.com/n0emis/ansible-role-bigbluebutton
-bbb_galaxy_role: n0emis.bigbluebutton
+# https://github.com/ebbba-org/ansible-role-bigbluebutton/releases
+bbb_role_ver: 609713bb5fccf9910b0a3545e789a8d75c62f9db
+bbb_galaxy_role: ebbba.bigbluebutton
+bbb_git_role: https://github.com/ebbba-org/ansible-role-bigbluebutton
+# Default presentation PDF file located in the files role directory
+bbb_default_pdf: default.pdf
```

Comment:
We have here the default presentation with a privacylx logo. If someone wants to / is reusing our role, they need to change this. I was trying to look at a way to store this "default.pdf" somehow in the inventory, but I didn't find a way. So I guess for the time being this is fine.

Reply:
Thanks for the review @francisco-core .
If someone would like to change the default pdf, they can change this variable in their inventory or during the run of the playbook.